samples/bluetooth/mesh/light_ctrl/src/main.c

```diff
@@ -56,7 +59,11 @@ static void isr_emds_cb(void *arg)
 
 #if defined(CONFIG_BT_CTLR)
 	/* Stop mpsl to reduce power usage. */
-	mpsl_lib_uninit();
+	int32_t err = mpsl_lib_uninit();
+
+	if (err != 0) {
+		printk("Could not stop MPSL (err %d)\n", err);
```

Comment:
Is there any point of doing this print in the interrupt, will the print be done before it shutdown. Maybe it it just as good doing a (void) on the mpsl_lib_uninit.

Reply:
It will print because I have added the `log_panic` which will send all pending logs through the backend and block before halting the system in the EMDS callback. There is a point in adding this I think, if this had been there from the beginning we wouldn't have had `mpsl_lib_uninit` in the sample doing nothing for two years, we would have seen the error message years ago. In general we should always error check things, I think.